torch/_inductor/codegen/common.py

```diff
@@ -1903,7 +1903,7 @@ def load(name: str, index: sympy.Expr) -> CSEVariable:
             @staticmethod
             def _update_store_cache(name: str, value: CSEVariable):
                 self.cse.store_cache[name] = value
-                if self.current_node:
+                if self.current_node and name in V.graph.buffers:
```

Comment:
Actually my mistake, this should be 
```suggestion
                if self.current_node and name in V.graph.name_to_buffer:
```

Reply:
oh.... yes, sorry for the mistake... Changed and please take a look again.